addons/account/views/account_move_views.xml

```diff
@@ -344,6 +344,7 @@
                     <filter string="Report Dates" name="date_between" domain="[('date', '&gt;=', context.get('date_from')), ('date', '&lt;=', context.get('date_to'))]" invisible="1"/>
                     <filter string="Report Dates" name="date_before" domain="[('date', '&lt;=', context.get('date_to'))]" invisible="1"/>
                     <separator/>
+                    <filter string="Analytic Accounts" name="analytic_accounts" domain="[('analytic_distribution', 'in', context.get('analytic_ids'))]" invisible="1"/>
```

Comment:
I don't know how important it is as the source term already exists, but you may have to add the occurence there in the pot.
You're in the team now, so you should know better than me :smile:

Reply:
Good catch indeed 😅 I updated the .pot file to include this occurrence (and also updated .pot before as a separate commit).